pandas/tests/arrays/sparse/test_indexing.py

```diff
@@ -13,13 +13,11 @@
 
 
 class TestGetitem:
-    def test_getitem(self):
-        def _checkit(i):
-            tm.assert_almost_equal(arr[i], arr.to_dense()[i])
```

Comment:
i think better to do the to_dense call once and then inline _checkit, but keep the loop (so just one test)